public/app/features/plugins/admin/pages/Browse.tsx

```diff
@@ -118,6 +123,19 @@ export default function Browse({ route }: GrafanaRouteComponentProps): ReactElem
                   ]}
                 />
               </div>
+
+              {/* Display mode */}
+              <div>
+                <RadioButtonGroup<PluginListDisplayMode>
+                  className={styles.displayAs}
+                  value={displayMode}
+                  onChange={setDisplayMode}
+                  options={[
+                    { value: PluginListDisplayMode.Table, icon: 'table', description: 'Display plugins in table' },
```

Comment:
nit: referring to this display mod as a table feels odd to me. It's more of a grid layout, no?

Reply:
Yeah, I think the naming is from me. I picked the same name as the icons. But now when you suggest it I think this makes sense.

Reply:
Yea, I think you are right, that feels more natural to me, too. 
I can change the name of the enum 👍